drivers/src/vertexai/index.ts

```diff
@@ -141,7 +141,13 @@ export class VertexAIDriver extends AbstractDriver<VertexAIDriverOptions, Vertex
             }));
         }
 
-        return models;
+        //Remove duplicates
+        const uniqueModels = Array.from(new Set(models.map(a => a.id)))
+            .map(id => {
+                return models.find(a => a.id === id) ?? {} as AIModel<string>;
+            })
```

Comment:
nit: could we use a map (record) where the key is the model id and the value is the model object? It sounds easier to understand to me